ibm/service/iamidentity/resource_ibm_iam_account_settings_template_assignment.go

```diff
@@ -0,0 +1,656 @@
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Assignment status.",
+			},
+			"resources": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "Status breakdown per target account of IAM resources created or errors encountered in attempting to create those IAM resources. IAM resources are only included in the response providing the assignment is not in progress. IAM resources are also only included when getting a single assignment, and excluded by list APIs.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"target": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Target account where the IAM resource is created.",
+						},
+						"account_settings": {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"resource_created": {
+										Type:        schema.TypeList,
+										MaxItems:    1,
+										Optional:    true,
+										Description: "Body parameters for created resource.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"id": {
+													Type:        schema.TypeString,
+													Optional:    true,
+													Description: "Id of the created resource.",
+												},
+											},
+										},
+									},
+									"error_message": {
+										Type:        schema.TypeList,
+										MaxItems:    1,
+										Optional:    true,
+										Description: "Body parameters for assignment error.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"name": {
+													Type:        schema.TypeString,
+													Optional:    true,
+													Description: "Name of the error.",
+												},
+												"error_code": {
+													Type:        schema.TypeString,
+													Optional:    true,
+													Description: "Internal error code.",
+												},
+												"message": {
+													Type:        schema.TypeString,
+													Optional:    true,
+													Description: "Error message detailing the nature of the error.",
+												},
+												"status_code": {
+													Type:        schema.TypeString,
+													Optional:    true,
+													Description: "Internal status code for the error.",
+												},
+											},
+										},
+									},
+									"status": {
+										Type:        schema.TypeString,
+										Required:    true,
+										Description: "Status for the target account's assignment.",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"history": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "Assignment history.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"timestamp": {
```

Comment:
Mark all inner fields of a computed attribute as computed.
It can't be (Requrired/optional)

Reply:
Done.